GUIDE.md

```diff
@@ -0,0 +1,165 @@
+# Assisted Installer Unofficial Install Guide
+
+## Prerequisites
+
+1. Install CentOS8/RHEL8 on the assisted installer host<sup id="a1">[1](#f1)</sup>
+   - This host will run minikube and the UI for deploying OpenShift on Bare Metal
+1. Setup DHCP/DNS records for the following OpenShift nodes and VIPs. List includes
+   - Master nodes
+   - Worker nodes
+   - API VIP
+   - Ingress VIP
+     > NOTE: dnsmasq can be used to setup [DHCP](https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#creating-dhcp-reservations-using-dnsmasq-option2_ipi-install-prerequisites) and [DNS](https://openshift-kni.github.io/baremetal-deploy/4.4/Deployment.html#creating-dns-records-using-dnsmasq-option2_ipi-install-prerequisites). Out of scope for this document but I've included links on how to do it.
+1. Ensure these values are properly set via nslookup and dig commands from your assisted installer host
+1. Download your Pull Secret from https://cloud.redhat.com/openshift/install/metal/user-provisioned
+
+## Deployment
+
+Procedure
+
+As `$USER` user with sudo privileges ,
+
+1.  Generate an SSH key
+
+        [$USER@assisted_installer ~]# ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ''
+
+1.  Install git and make on your CentOS assisted installer host if not already present
+
+        [$USER@assisted_installer ~]# dnf install -y make git
+
+1.  Create a directory under `~/` labeled `assisted-installer` and `git clone` the `test-infra` git repository
+
+        [$USER@assisted_installer ~]# mkdir ~/assisted-installer
+        [$USER@assisted installer assisted-installer]# cd ~/assisted-installer/
+        [$USER@assisted_installer assisted-installer]# git clone https://github.com/openshift/assisted-test-infra
+        [$USER@assisted_installer assisted-installer]# cd assisted-test-infra
+
+1.  Setup the assisted installer's environment by running the `install_environment.sh` script
+
+        [$USER@assisted_installer assisted-test-infra]# ./scripts/install_environment.sh
```

Comment:
no need. just run "make create_full_environment" and move make to the line where git is installed.

Reply:
so after the install make, git, just run 'make create_full_environment' right?

Reply:
right